Artesian/Artesian.SDK/Service/Clients/Exceptions/Client/ArtesianSdkProblemDetail.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+
+namespace Artesian.SDK.Service
+{
+    /// <summary>
+    /// Problem Deatail
+    /// </summary>
+    public class ArtesianSdkProblemDetail : Exception
```

Comment:
should not derive from Exception.

Reply:
How would I pass the problem details otherwise? 
public ArtesianSdkRemoteException(string message)
            : base(message)
        {
        }

will not pass problemDetail so I passed it through as an innerException, is there a way around this?

 public ArtesianSdkRemoteException(string message, Exception innerEx)
            : base(message, innerEx)
        {
        }

Reply:
changing the Exception class :-)

Add a parameter in the ctor and add a get-only property.